src/views/index.ecr

```diff
@@ -181,9 +181,10 @@
             <tr><td><code>[^abc]</code></td><td>any character except a, b, or c</td></tr>
             <tr><td><code>[a-z]</code></td><td>any character from range a-z</td></tr>
             <tr><td><code>[a-z0-9]</code></td><td>any character from range a-z or 0-9</td></tr>
+            <tr><td><code>[a|b]</code></td><td>a or b</td></tr>
             <tr><td><code>.</code></td><td>any character</td></tr>
             <tr><td><code>(...)</code></td><td>group capture</td></tr>
-            <tr><td><code>(a|b)</code></td><td>a or b</td></tr>
+            <tr><td><code>(a|b)</code></td><td>capture a or b</td></tr>
```

Comment:
how do you feel about this?
```suggestion
            <tr><td><code>(a|b)</code></td><td>a or b (with capture)</td></tr>
            <tr><td><code>(?:a|b)</code></td><td>a or b (without capture)</td></tr>
```

Reply:
Looks much better and is more consistent with the other "a or b".